docs/reference/mobile/android/setup.md

```diff
@@ -2,36 +2,88 @@
+1. In your Unity Editor, add the MonoBehaviour script to your mobile client-worker GameObject.
+
+<%(/Expandable)%>
+
+If you want to launch your application on a physical device, you need to 
+
+1. Ensure you have USB debugging enabled on it. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. If you want to connect it to a local deployment, ensure your computer and your mobile device are connected to the same network.
+
+### Prepare your project to connect to a local deployment
+
+To connect your Android device to a local deployment, you need to prepare its configuration. Follow the steps below to ensure your project is set up correctly:
+
+1. Open your project in your Unity Editor.
+1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
+1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
+1. Select **Save** and close the window.
+
+### Prepare your project to connect to a cloud deployment
+
+To connect your Android device to a cloud deployment, you need to authenticate against our services. This guide describes how to authenticate using the [development authentication flow](https://docs.improbable.io/reference/latest/shared/auth/development-authentication) which we provide for the early stages of game development. The GDK for Unity provides tooling around it to help you iterate faster on your mobile application.
+
+1. Open your project in your Unity Editor.
+1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
+1. In the **Dev Auth Token Settings** specify the lifetime of the token and the path to the [Resources folder](https://unity3d.com/learn/tutorials/topics/best-practices/resources-folder) that you would like to store the generated token in.
+1. Select **Save** and close the window.
+1. Go to **SpatialOS** > **Generate Dev Authentication Token**. This generates a `DevAuthToken.txt` asset in the folder you specified in the configuration window. If your worker connector inherits from the `DefaultMobileWorkerConnector` script, it will automatically read that token when running the application and authenticate against our services.
```

Comment:
```suggestion
1. Select **SpatialOS** > **Generate Dev Authentication Token**. This generates a `DevAuthToken.txt` asset in the folder you specified in the configuration window. If your worker connector inherits from the `DefaultMobileWorkerConnector` script, it will automatically read that token when running the application and authenticate against our services.
```

Reply:
> If your worker connector inherits from the `DefaultMobileWorkerConnector` script, it will automatically read that token when running the application and authenticate against our services.

Could this go in a `>` block?

Reply:
Also what should they do if the don't inherit from that script?

Reply:
then they need to implement it on their own?

Reply:
Probably want to tell them that then 😉